packages/react-native-worklets/android/src/main/java/com/swmansion/worklets/ScriptBufferWrapper.kt

```
@@ -58,45 +61,73 @@ class ScriptBufferWrapper(
-                    content = readBytes(connection.inputStream)
+        private fun downloadScriptToFile(
+            url: String,
+            outputFile: File,
+        ) {
+            val connection = URL(url).openConnection() as HttpURLConnection
+            try {
+                val statusCode =
+                    try {
+                        connection.responseCode
+                    } catch (e: IOException) {
+                        throw RuntimeException("[Worklets] Could not connect to development server.\n", e)
```

detekt (1.23.8)

[warning] RuntimeException is a too generic Exception. Prefer throwing specific exceptions that indicate a specific error case.

(detekt.exceptions.TooGenericExceptionThrown)